src/store/actionTypes.tsx

```diff
@@ -0,0 +1,12 @@
+export enum ActionType {
```

Comment:
we don't usually create object for types, it's redundant nesting
just export action types right away as you did before